### PR TITLE
support remapIdentity simple policies

### PR DESCRIPTION
**Pull request by sjhx**
reflect the remapIdentity support recently added to https://github.com/containers/image

Signed-off-by: Stuart Hayton <stuart.hayton@uk.ibm.com>

resolves: https://github.com/IBM/portieris/issues/92

### CODE REVIEW

File: pkg/apis/securityenforcement/client/clientset/versioned/clientset.go [hunk 1]
```diff
@@ -53,14 +51,9 @@ func (c *Clientset) Discovery() discovery.DiscoveryInterface {
 }
 
 // NewForConfig creates a new Clientset for the given config.
-// If config's RateLimiter is not set and QPS and Burst are acceptable,
-// NewForConfig will generate a rate-limiter in configShallowCopy.
 func NewForConfig(c *rest.Config) (*Clientset, error) {
 	configShallowCopy := *c
 	if configShallowCopy.RateLimiter == nil && configShallowCopy.QPS > 0 {
-		if configShallowCopy.Burst <= 0 {
```
Comment (jhart1685) [inline pkg/apis/securityenforcement/client/clientset/versioned/clientset.go:61]: Interested in why this generated code has changed?

**Review by jhart1685 — DISMISSED** [review]
Mostly looks good to me.
But I think some docs / example policy ought to be in this PR too.
Ideally, we should also have some new e2e tests, but I think those can be added separately.

File: pkg/apis/securityenforcement/client/clientset/versioned/clientset.go (hunk 1 above)
Comment (sjhx) [reply pkg/apis/securityenforcement/client/clientset/versioned/clientset.go:61]: It looks like it was unnecessary to regenerate the code so merging this change is optional, curious about this change too, my thinking is that I followed the documented process with up to date code generator so likely that future dode gens will do the same. Will look at blame for said lines.

Comment (sjhx) [reply pkg/apis/securityenforcement/client/clientset/versioned/clientset.go:61]: looks like these lines were introduced here: https://github.com/IBM/portieris/pull/111, they appear to be connected to rate limiting calls to the apiserver acording to configuration, I dont believe they are needed and seem to be unable to have them generated to propose we go with their removal

**Review by bainsy88 — APPROVED** [review]
LGTM

**Pull request merged**